LEAF_Request_Portal/templates/reports/LEAF_import_data.tpl

```diff
@@ -169,37 +270,54 @@
             .appendTo(row);
 
         var columnSelect = $(document.createElement('td'))
-            .append(buildSheetSelect(indicator.indicatorID, sheet_data))
+            .append(buildSheetSelect(indicator.indicatorID, sheet_data, indicator.required))
             .appendTo(row);
 
         return row;
     }
 
-    function doStuff(initiator, requestData) {
+    function doStuff(categoryID, initiator, requestData) {

```

Comment:
Rename this function to something that describes what it does.

Reply:
Addressed in commit `ddf`